crates/core/src/task_center_types.rs

```diff
@@ -80,6 +80,10 @@ pub enum TaskKind {
     /// A background task that the system needs for its operation. The task requires a system
     /// shutdown on errors and the system will wait for its graceful cancellation on shutdown.
     BifrostBackgroundHighPriority,
+    /// A background appender. The task will log on errors but the system will wait for its
+    /// graceful cancellation on shutdown.
+    #[strum(props(OnCancel = "wait", OnError = "log"))]
+    BifrostAppender,
```

Comment:
Since we are awaiting on cancel but do not listen to the task's cancellation token, we assume that this task will always be canceled together with the task that holds the `AppenderHandle`, right?

Reply:
Yes, that's correct. appender handle's description explains this.